packages/graphql_server/src/api/twitterScrapingService.ts

```diff
@@ -0,0 +1,114 @@
+import axios, { AxiosRequestConfig } from 'axios'
+import {
+  TwitterPost,
+  TwitterUser,
+  TwitterProfileResponse
+} from '../../types/twitterScrapingService'
+
+const username = 'typescript2023'
+const apiKey = process.env.SCRAPING_BOT_API_KEY || ''
+const apiEndPoint = 'http://api.scraping-bot.io/scrape/data-scraper'
```

Comment:
Store all of these 3 constants as environment variables - if we ever need to change them, we shouldn't have to go through the code to do that